frontend/src/main/components/Courses/CourseForm.js

```diff
@@ -14,6 +16,29 @@ function CourseForm({ initialCourse, submitAction, buttonLabel = "Create" }) {
 
```

Comment:
Please replace `// Stryker enable all` with `// Stryker restore all`  

This may result in additional Stryker mutations surviving; at that point, we'll need to either fix those, or use a more surgical approach to excluding code from Stryker.  At the moment, the entire rest of this file is being excluded because `// Stryker enable all` is not a thing.   So the `disable all` on line 7 is taking effect over the entire file.

I know it may seem unfair that you are being asked to fix something that wasn't your fault.   This happens in the real world all the time in large code bases, and it sucks, but it comes with the territory, so get used to it I guess :-)

Reply:
I'm ok to make this a separate PR.

Reply:
Sounds good, I'll start another PR fixing this then. In addition, would you like me to do this for all the files or just the ones modified in this PR?

Reply:
Actually never mind, I just saw the other PR you created. Thanks!